sdk/ml/azure-ai-ml/azure/ai/ml/operations/_batch_endpoint_operations.py

```diff
@@ -238,6 +239,10 @@ def invoke(
             self._validate_deployment_name(endpoint_name, deployment_name)
 
         if input and isinstance(input, Input):
+            warnings.warn(
+                "The property 'input' is deprecated. Please use 'inputs' instead.",
+                DeprecationWarning,
```

Comment:
The if-statement implies that the property `input` is only deprecated if it's an instance of type `Input` - is that the case?Or could it have never been any other type? (in which case, why were we checking the type in the first place?)

Reply:
Good catch. Will make the appropriate change

Reply:
Actually after further review, input as instance of Input is the only place in invoke when we use input, even though there is a "str" option listed as the doc string. I am not sure why that is but I will keep it as is.